lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -381,6 +381,21 @@ private VerifiableCredentialBundle getVerifiableCredentials(
                                 evcsIdentityVcs,
                                 isPendingEvcs,
                                 vcBundle.isF2fIdentity());
+
+                if (hasPartiallyMigratedVcs) {
+                    var vcs = vcBundle.credentials;
```

Comment:
will here vcs be initialised to tacticalVcs ?

Reply:
depends if EVCS_READ_ENABLED is true. Maybe I should be using tacticalVCs instead